src/routes/+layout.svelte

```diff
@@ -36,9 +38,7 @@
   <Banner />
   <div class="usa-overlay" />
   <Header {navItems} {secondaryNavItems} {siteTitle} bind:navMenuExpanded />
-  <main id="main-content">
```

Comment:
We had a few occurences of providing multiple `main` tags on some pages, so I opted to remove this and leave it up to each route's layout / page to handle adding the tag where appropriate (some need it to wrap everything, while others that have breadcrumbs and sidenavs might want it only wrapping the actual content of the page). This change obviously means that it looks like there's a lot more shifting here than there actually is; I'd suggest viewing the route files [with whitespace changes ignored](https://github.com/la-ldaf/ldaf-site/pull/196/files?w=1).